Add language files (Simplified Chinese and Traditional Chinese)

snowwolf725 opened the pull request:
Add language files
zh_CN : Simplified Chinese
zh_TW : Traditional Chinese
In order to read lang files which with 5 charatacters, I also modified the zmb.sh.

On [POSIX](https://en.wikipedia.org/wiki/POSIX) platforms such as [Unix](https://en.wikipedia.org/wiki/Unix), [Linux](https://en.wikipedia.org/wiki/Linux) and others, locale identifiers are defined by [ISO/IEC 15897](https://en.wikipedia.org/wiki/ISO/IEC_15897), which is similar to the [BCP 47](https://en.wikipedia.org/wiki/BCP_47) definition of language tags, but the locale variant modifier is defined differently, and the [character set](https://en.wikipedia.org/wiki/Character_set) is included as a part of the identifier.[[citation needed](https://en.wikipedia.org/wiki/Wikipedia:Citation_needed)] It is defined in this format: [language[_territory][.codeset][@modifier]]. (For example, [Australian English](https://en.wikipedia.org/wiki/Australian_English) using the [UTF-8](https://en.wikipedia.org/wiki/UTF-8) encoding is en_AU.UTF-8.)

Ref:
https://en.wikipedia.org/wiki/Locale_(computer_software)
https://www.debian.org/doc/manuals/debian-reference/ch08.en.html

snowwolf725 committed:
Add language files (Simplified Chinese and Traditional Chinese)
Modify zmb.sh to support read lang files which with 5 charatacters

grm34 commented:
xD thanks again  but look: the goal here is (for example) that  `en_US.UTF-8` and `en_UK.UTF-8` and `en_AU.UTF-8`  got English translation, I'm not gonna translate each variant.

The modif you made to `zmb` makes all translations bugs, it will check for variant in filename and none match.

Is both Chinese variants really needed here? If so rename to get `zh.cfg` and `chi.cfg` as I can merge the PR, I will modify that will need to be once pushed. Use `zh` for the most used variant. But I think main chinese trad is enough, zmb checks for `zh` and all variants will got chinese trad.

snowwolf725 commented:
The modify with failover mechanism in script, it will both check lang files with two characters  and five characters.
So  en_US.UTF-8, en_UK.UTF-8, and en_AU.UTF-8 got English translation.
You don't need to handle each variant.
It works fine in my environment.

14 hundred million people use Simplified Chinese, but only 24 million people use Traditional Chinese.
So Traditional Chinese is most popular in Chinese world.
I prefer use Simplified Chinese to zh.cfg, if you only want keep zh as Chinese language file.
However, there are slight differences in the usage between Traditional Chinese and Simplified Chinese.
For example, the sport known as football in Great Britain is usually called soccer in the United States.
![Example](https://i.ibb.co/jHr0ZDk/example.png)

snowwolf725 commented:
Maby we need three language files to handle all Chinese environment.
zh_CN : Simplified Chinese (Use in China, Hongkong, Singapore, Malaysia)
zh_TW : Traditional Chinese (Use in Taiwan, Hongkong, Singapore, Malaysia)
zh:  Simplified Chinese 

Ref:
https://en.wikipedia.org/wiki/Malaysian_Mandarin
https://en.wikipedia.org/wiki/Singaporean_Mandarin
